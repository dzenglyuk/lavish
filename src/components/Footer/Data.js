export const infoLinks = [
    {
        title: "About Us",
        links: [
            {
                name: "How it works",
                value: "/sign-up",
            },
            {
                name: "Testimonials",
                value: "/",
            },
            {
                name: "Careers",
                value: "/",
            },
            {
                name: "Investors",
                value: "/",
            },
            {
                name: "Terms of Service",
                value: "/",
            },
        ],
    },
    {
        title: "Contact Us",
        links: [
            {
                name: "Contact",
                value: "/",
            },
            {
                name: "Support",
                value: "/",
            },
            {
                name: "Destinations",
                value: "/",
            },
            {
                name: "Sponsorships",
                value: "/",
            },
        ],
    },
];

export const socialLinks = [
    {
        title: "Videos",
        links: [
            {
                name: "Submit Video",
                value: "/",
            },
            {
                name: "Ambassadors",
                value: "/",
            },
            {
                name: "Agency",
                value: "/",
            },
            {
                name: "Influencer",
                value: "/",
            },
        ],
    },
    {
        title: "Social Media",
        links: [
            {
                name: "Instagram",
                value: "/",
            },
            {
                name: "Facebook",
                value: "/",
            },
            {
                name: "Youtube",
                value: "/",
            },
            {
                name: "Twitter",
                value: "/",
            },
        ],
    },
];

export const subscription = {
    heading: "Join our exclusive membership to receive the latest news and trends",
    text: "You can unsubscribe at any time."
};