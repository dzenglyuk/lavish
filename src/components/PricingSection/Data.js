const starterAccount = {
    info: {
        name: "Starter",
        price: "$8.99",
        period: "per month"
    },
    features: ["100 Transactions", "2% Cash Back", "$10,000 Limit"]
};

const goldAccount = {
    info: {
        name: "Gold",
        price: "$29.99",
        period: "per month"
    },
    features: ["1000 Transactions", "3.5% Cash Back", "$100,000 Limit"]
};

const diamondAccount = {
    info: {
        name: "Diamond",
        price: "$99.99",
        period: "per month"
    },
    features: ["Unlimited Transactions", "5% Cash Back", "Unlimited Spending"]
};

export {
    starterAccount,
    goldAccount,
    diamondAccount
};