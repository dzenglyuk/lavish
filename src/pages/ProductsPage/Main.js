import React from "react";
import HomeSection from "../../components/HomeSection/Main";
import { homeObjTwo, homeObjThree } from "../Data"; 

const ProductsPage = () => {
    return (
        <React.Fragment>
            <HomeSection {...homeObjTwo} />
            <HomeSection {...homeObjThree} />
        </React.Fragment>
    )
}

export default ProductsPage;
