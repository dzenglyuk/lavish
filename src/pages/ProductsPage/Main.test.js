import React from "react";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import { StaticRouter as Router } from "react-router";

import { homeObjTwo, homeObjThree } from "../Data";
import ProductsPage from "./Main";

test("Renders correctly", () => {
    const { getByText } = render(
        <Router>
            <ProductsPage />
        </Router>
    );
    getByText(homeObjTwo.headLine);
    getByText(homeObjTwo.topLine);
    getByText(homeObjThree.headLine);
    getByText(homeObjThree.topLine);
});

test("Matches the snapshot", () => {
    const tree = renderer
        .create(
            <Router>
                <ProductsPage />
            </Router>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
