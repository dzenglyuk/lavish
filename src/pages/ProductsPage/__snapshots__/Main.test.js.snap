// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
Array [
  <div
    className="home__section"
  >
    <div
      className="container"
    >
      <div
        className="row home__home-row"
        style={
          Object {
            "display": "flex",
            "flexDirection": "row-reverse",
          }
        }
      >
        <div
          className="col"
        >
          <div
            className="home__text-wrapper"
          >
            <div
              className="top-line"
            >
              EASY SETUP
            </div>
            <h1
              className="heading dark"
            >
              Super fast and simple onboarding process
            </h1>
            <p
              className="home__subtitle dark"
            >
               
              Get everything set up and ready in under 10 minutes. All you need to do is add your information and you're ready to go.
               
            </p>
            <a
              href="/sign-up"
              onClick={[Function]}
            >
              <button
                className="btn 
			btn--primary
			btn--wide
			blue"
              >
                Start Now
              </button>
            </a>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="home__img-wrapper"
          >
            <img
              alt="Vault"
              className="home__img"
              src="images/svg-7.svg"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="home__section darkBg"
  >
    <div
      className="container"
    >
      <div
        className="row home__home-row"
        style={
          Object {
            "display": "flex",
            "flexDirection": "row",
          }
        }
      >
        <div
          className="col"
        >
          <div
            className="home__text-wrapper"
          >
            <div
              className="top-line"
            >
              100% SECURE
            </div>
            <h1
              className="heading"
            >
              Stay protected 24/7 anywhere anytime
            </h1>
            <p
              className="home__subtitle"
            >
               
              We have you covered no matter where you are located. Over 140 locations worldwide to ensure you have access anytime
               
            </p>
            <a
              href="/sign-up"
              onClick={[Function]}
            >
              <button
                className="btn 
			btn--primary
			btn--wide
			blue"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="home__img-wrapper"
          >
            <img
              alt="Vault"
              className="home__img"
              src="images/svg-5.svg"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
